db_test.go

```diff
@@ -1053,8 +1053,8 @@ func TestDB_Stats(t *testing.T) {
 	stats := db.Stats()
 	if stats.TxStats.GetPageCount() != 2 {
 		t.Fatalf("unexpected TxStats.PageCount: %d", stats.TxStats.GetPageCount())
-	} else if stats.FreePageN != 0 {
-		t.Fatalf("unexpected FreePageN != 0: %d", stats.FreePageN)
+	} else if stats.FreePageN != 2 {
```

Comment:
I think I've messed up the stats, so expect this to go back to its original value